### PR TITLE
Qa fixes misc

### PR DESCRIPTION
**Pull request by luccasmmg**

### CODE REVIEW

File: deployment/frontend/next.config.mjs [hunk 1]
```diff
@@ -12,6 +12,8 @@ const config = {
     ignoreDuringBuilds: true,
   },
   reactStrictMode: true,
+  output: "standalone",
```
Comment (steveoni) [inline deployment/frontend/next.config.mjs]: does `output: "standalone",` works with `npm run start` use to think this creates a server.js that should be run with `npm server.js` or so

File: e2e-tests/cypress/screenshots/dashboard.cy.js/Dashboard Test -- Should test teams page (failed).png
Comment (steveoni) [inline e2e-tests/cypress/screenshots/dashboard.cy.js/Dashboard Test -- Should test teams page (failed).png:1]: you can remove this test result.

**Review by steveoni — APPROVED** [review]
lgtm!

File: deployment/frontend/next.config.mjs (hunk 1 above)
Comment (luccasmmg) [reply deployment/frontend/next.config.mjs]: true, my mistake, forgot to checkout that file

**Pull request merged**